index.html

```diff
@@ -34,22 +38,212 @@ <h1>Hey There.
+                    <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+                    <ul class="btn-hold">
+                        <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                        <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                        <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+                    </ul>
+                    </div>
+                    <div class="actz">
+                        <button class="clicks btnn">See Project</button>
+                    </div>
+                    
+                </div>
+        </div>
+
+
+        <div class="card-works">
+            <div class="place-hold">
+                <div class="r-b">
+                    <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+                <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+                <ul class="btn-hold">
+                    <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                    <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                    <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+                </ul>
+                </div>
+                <div class="actz">
+                    <button class="clicks btnn">See Project</button>
+                </div>
+                
+            </div>
+    </div>
+
+    <div class="card-works">
+        <div class="place-hold">
+            <div class="r-b">
+                <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+            <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+            <ul class="btn-hold">
+                <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+            </ul>
+            </div>
+            <div class="actz">
+                <button class="clicks btnn">See Project</button>
+            </div>
+            
+        </div>
+</div>
+          </div>
+    </div>   
+
+
+    
+    
+        </section>
+
+        <!--about myself-->
+        <section class="about">
+        
+            <h1 class="abt-txt1">About me</h1>
+            <h4 class="abt-txt2">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
```

Comment:
Please ensure that the paragraph (e.g., "I'm a software ...") is a `p` tag according to the project requirement.